.github/workflows/rebuild_org.yml

```diff
@@ -5,7 +5,7 @@ on:
   repository_dispatch:
     types: [run_tmate, run]
   schedule:
-    - cron: "0 * * * *"
```

Comment:
So another option here would be a range. We could keep the existing `20` rebuilds limit and run for example `0 0-4 * * *`. Thoughts on running over a constrained period of time instead of front-loading as hard with a large rebuild limit?

Reply:
I'm fine with that.  Please see commit ff59e5f.